plugins/user-authenticators/ldap/src/main/java/org/apache/cloudstack/ldap/LdapConfiguration.java

```diff
@@ -65,7 +65,7 @@ public class LdapConfiguration implements Configurable{
             String.class,
             "ldap.user.memberof.attribute",
             "memberof",
-            "the reverse membership attibute for group members",
+            "the reverse membership attribute for group members",
```

Comment:
this one will not update on existing installations and should (very striuctly speaking) have an update statement in the schema upgrade file.